portlets/src/main/frontend/src/apps/profileStats/profilStatsAPI.js

```diff
@@ -69,8 +69,8 @@ export function getSpaces() {
   })
 }
 
-export function getSpacesOfUser() {
-  return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/v1/social/users/${eXo.env.portal.profileOwner}/spaces`, {
+export function getSpacesOfUser(offset, limit) {
+  return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/v1/social/users/${eXo.env.portal.profileOwner}/spaces?offset=${offset || 0}&limit=${limit|| 0}&returnSize=true`, {
```

Comment:
Make the limit greater than 0, i.e 10 otherwise you won't get any results if limit is not set

Reply:
done !